### PR TITLE
Base in error handling

### PR DESCRIPTION
**Pull request by yminsky**
Pretty simple; had to open Stdio in a few places, and some other minor tweaks.

### CODE REVIEW

File: book/07-error-handling.html [hunk 1]
```diff
@@ -181,9 +181,9 @@ <h2>Error and Or_error</h2>
 
           <p>
 	    S-expressions are supported by the Sexplib package that
-            is distributed with Core and is the most common
-            serialization format used in Core. Indeed, most types in
-            Core come with built-in s-expression converters.
+            is distributed with Base and is the most common
```
Comment (avsm) [inline book/07-error-handling.html]: forward reference to Part II and the serialization chapter?

File: book/07-error-handling.html [hunk 2]
```diff
@@ -446,7 +446,7 @@ <h1>Declaring Exceptions Using <code>[@@deriving sexp]</code></h1>
           <h2>Helper Functions for Throwing Exceptions</h2>
 
           <p>
-	    OCaml and Core provide a number of helper functions to
+	    OCaml and Base provide a number of helper functions to
```
Comment (avsm) [inline book/07-error-handling.html]: Just a nit here, but is "OCaml and Base" necessary, as opposed to just "Base"?  Anything that OCaml provides is re-exported by Base, often with different conventions, so adding OCaml here seems redundant.

File: book/07-error-handling.html (hunk 1 above)
Comment (yminsky) [reply book/07-error-handling.html]: There's already a forward reference a few lines down.

File: book/07-error-handling.html (hunk 2 above)
Comment (yminsky) [reply book/07-error-handling.html]: Fixed.

**Pull request merged**